docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -69,6 +93,8 @@ \section{Critical Assumptions}
 	you could assume a part will never fail, but it is generally better to include
 	this potential failure mode.}
 
+There are no critical assumptions made.
```

Comment:
> Under critical assumptions you can list assumptions that restrict how much you have to mitigate and eliminate hazards. For instance, you can assume that the operator of your machine isn’t intentionally trying to misuse it.

I feel like there should be something here. Also remove Dr. Smith's comment above this.